### PR TITLE
Themes - Tukan /Cicada - Page content main section view fix

### PR DESCRIPTION
**Pull request by opnsenseuser**
this was the problem:
grafik

now its fixed for both themes:
grafik

### CODE REVIEW

File: misc/theme-cicada/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-cicada
-PLUGIN_VERSION=		1.14
+PLUGIN_VERSION=		1.14_2
```
Comment (fichtner) [inline misc/theme-cicada/Makefile]:
```suggestion
PLUGIN_VERSION=		1.14
```

File: misc/theme-cicada/Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 PLUGIN_REVISION=	1
```
Comment (fichtner) [inline misc/theme-cicada/Makefile]:
```suggestion
PLUGIN_REVISION=	2
```

File: misc/theme-tukan/Makefile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-tukan
-PLUGIN_VERSION=		1.13
+PLUGIN_VERSION=		1.13_2
```
Comment (fichtner) [inline misc/theme-tukan/Makefile]:
```suggestion
PLUGIN_VERSION=		1.13
```

File: misc/theme-tukan/Makefile [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
 PLUGIN_REVISION=	1
```
Comment (fichtner) [inline misc/theme-tukan/Makefile]:
```suggestion
PLUGIN_REVISION=	2
```

**Comment by opnsenseuser** [comment]
Thx 👍

**Comment by fichtner** [comment]
Merged, thanks! One revision change missing, will add manually.

**Comment by fichtner** [comment]
Note this will not be part of 19.1.4 tomorrow. Builds have already been finished.

**Comment by opnsenseuser** [comment]
Ok

**Pull request merged**